frontend/src/metabase/actions/components/ActionViz/ActionDashcardSettings.unit.spec.tsx

```diff
@@ -456,6 +456,16 @@ describe("ActionViz > ActionDashcardSettings", () => {
     ).toBeInTheDocument();
   });
 
+  it("should be valid and not crash when the action does not have parameters", async () => {
+    const { closeSpy } = setup({
+      dashcard: createMockActionDashboardCard({
+        action: createMockQueryAction(),
```

Comment:
:x: This test passes in `master`

```suggestion
        action: createMockQueryAction({ parameters: undefined }),
```